### PR TITLE
Solved the issue for reading BQ's DATETIME for time prior to 1970-01-01

### PR DESCRIPTION
**Pull request by himanshukohli09**

### CODE REVIEW

File: connector/third_party/apache-spark/src/main/java/com/google/cloud/spark/bigquery/ArrowSchemaConverter.java [hunk 1]
```diff
@@ -468,11 +468,21 @@ final long getLong(int rowId) {
     @Override
     final UTF8String getUTF8String(int rowId) {
       long epoch = accessor.get(rowId);
+      long seconds = epoch / 1000000;
```
Comment (davidrabinowitz) [inline connector/third_party/apache-spark/src/main/java/com/google/cloud/spark/bigquery/ArrowSchemaConverter.java:471]: Please convert to the magic numbers to constants (`ONE_MILLION`, `ONE_BILLION`)

**Review by davidrabinowitz — CHANGES_REQUESTED** [review]

**Review by davidrabinowitz — CHANGES_REQUESTED** [review]
Please add a unit test

File: connector/third_party/apache-spark/src/main/java/com/google/cloud/spark/bigquery/ArrowSchemaConverter.java (hunk 1 above)
Comment (himanshukohli09) [reply connector/third_party/apache-spark/src/main/java/com/google/cloud/spark/bigquery/ArrowSchemaConverter.java]: Done

File: connector/third_party/apache-spark/src/main/java/com/google/cloud/spark/bigquery/ArrowSchemaConverter.java [hunk 2]
```diff
@@ -454,6 +454,9 @@ final long getLong(int rowId) {
   private static class TimestampMicroVectorAccessor extends ArrowSchemaConverter.ArrowVectorAccessor {
 
     private final TimeStampMicroVector accessor;
+    private static final int ONE_MILLION = 1_000_000;
```
Comment (davidrabinowitz) [inline connector/third_party/apache-spark/src/main/java/com/google/cloud/spark/bigquery/ArrowSchemaConverter.java:457]: nit: Please order the constants according to size

**Review by davidrabinowitz — APPROVED** [review]

**Pull request merged**